eservices-monitoring-exporter/src/main/scala/it/pagopa/interop/eservicesmonitoringexporter/util/Utils.scala

```diff
@@ -1,39 +1,22 @@
 package it.pagopa.interop.eservicesmonitoringexporter.util
 
-import it.pagopa.interop.catalogmanagement.{model => DependencyCatalog}
 import it.pagopa.interop.eservicesmonitoringexporter.model._
-import it.pagopa.interop.eservicesmonitoringexporter.model.State.{ACTIVE, INACTIVE}
-import it.pagopa.interop.eservicesmonitoringexporter.model.Technology.{REST, SOAP}
 
 object Utils {
 
-  implicit class TechnologyWrapper(private val tech: DependencyCatalog.CatalogItemTechnology) extends AnyVal {
-    def toApi: Technology = tech match {
-      case DependencyCatalog.Rest => REST
-      case DependencyCatalog.Soap => SOAP
-    }
-  }
-
-  implicit class StateWrapper(private val tech: DependencyCatalog.CatalogDescriptorState) extends AnyVal {
-    def toApi: State = tech match {
-      case DependencyCatalog.Published  => ACTIVE
-      case DependencyCatalog.Deprecated => ACTIVE
-      case _                            => INACTIVE
-    }
-  }
```

Comment:
you lost this conversion.
Let's keep `Technology` and `State` types and keep using the `toString` [here](https://github.com/pagopa/interop-be-jobs/pull/113/files?diff=split&w=1#diff-abd9af814662e1146a38d7d4a4148f52b411d1a91f6e01318dee88de6393bfd7L32-L33)